msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsAllAuthoredMsgForOneUser.jsp

```diff
@@ -226,9 +226,18 @@
 	  					<h:outputText escape="false" value="#{stat.message}"/>
 					<f:verbatim>
 	  					</span>
-		  				<h4 style="border-bottom:1px solid #ccc;padding-bottom:5px;overflow:hidden">
-						<span><img src="/library/image/silk/table_add.png" />&nbsp;</f:verbatim><h:outputText value="#{msgs.cdfm_message_count}" /><f:verbatim>:&nbsp;<span  id="wordCountSpan</f:verbatim><h:outputText value="#{stat.msgId}"/><f:verbatim>"> </span></span>
+						<span>
+							<img src="/library/image/silk/table_add.png" />&nbsp;</f:verbatim>
```

Comment:
Could we consider using font awesome for this instead of a silk icon?

![image](https://user-images.githubusercontent.com/8653754/171380151-6e77edeb-41d8-4b36-8ce1-741e89dead2b.png)